etl/exportConsumer.go

```diff
@@ -0,0 +1,36 @@
+package etl
```

Comment:
Not camelcase for filenames, so etl/exportconsumer.go.

Reply:
all filenames have been renamed